Retrieve either SequencingReadsCollection or UploadedPathogenGenome for sampleview

ttung opened the pull request:
### Description
Samples can be one or the other.  Retrieve both, and fill in the result with whichever one is appropriate.

Depends on #128, #129, #130

### Test plan
pytest

phoenixAja reviewed (approved):
cool cool cool!!! thanks so much for your help on this!!

Commit:
Retrieve either SequencingReadsCollection or UploadedPathogenGenome for sampleview

### Description
Samples can be one or the other.  Retrieve both, and fill in the result with whichever one is appropriate.

### Test plan
pytest?
